pages/common/csvkit.md

```diff
@@ -0,0 +1,21 @@
+# csvkit
+
+> Manipulation toolkit for CSV files.
+> For more information see commands: `csvclean`, `csvcut`, `csvformat`, `csvgrep`, `csvlook`, `csvpy`, `csvsort`, `csvstat`.
+> More information: <https://csvkit.readthedocs.io/en/0.9.1/cli.html>.
+
+- Run a command on a CSV file with the non default delimiter:
```

Comment:
Perhaps this will read better?
```suggestion
- Run a command on a CSV file with a custom default delimiter:
```

Also, this seems like a more advanced usage of the command. Perhaps we could include a basic one first? Generally, the examples on a tldr-page should go in the order that you might use them in.

Also, what does the `csvkit` command actually do? You mention running it on a CSV file, but don't mention what it would actually do to the CSV file.

Reply:
`csvkit` is not a command, but a kit of several commands which have several common options this page serves as a reference to all of them as it was suggested in the issue #749

Reply:
Ah, I see. In that case it's probably fine :-)